docs/Installation/Installation.md

```diff
@@ -143,10 +143,9 @@ To build with the docker image:
          SPECTRE_ROOT
    ```
    When cmake configuration is done, you are ready to build target executables.
-   Compile the code with `make -jN` where `N` is the number of cores to build on
```

Comment:
Can you squash this fixup into the appropriate commits? You'll probably need to split this commit and then squash each part separately. GitHub and Atalassian both have documentation on how to do this: https://docs.github.com/en/github/using-git/about-git-rebase but please ask if you are unsure. You can "undo" the most recent commit with `git reset --soft HEAD~` and then commit each hunk separately. At that point you can squash the two small commits into the appropriate commits